src/Microsoft.PowerShell.Commands.Utility/commands/utility/AddMember.cs

```diff
@@ -11,6 +11,7 @@
 using System.Management.Automation.Host;
 using System.Management.Automation.Internal;
 using System.Reflection;
+using AllowNullAttribute = System.Management.Automation.AllowNullAttribute;
```

Comment:
I think we can use new System.Diagnostics.CodeAnalysis.AllowNullAttribute

Reply:
Would there be an advantage, I guess we cannot remove the existing attribute due to it being public. Could this break edge cases where e.g. people analyze attributes on the AST?

Reply:
Let's not make this change in this PR

Reply:
I recommend to remove the unnecessary `[SuppressMessage("xxxx", "xxxxxx")]` from those affected files so that we can remove the `using System.Diagnostics.CodeAnalysis;` directive. Then `AllowNullAttribute` won't be ambiguous.

Reply:
@bergmeister Can you please address my comment? Thanks!